pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -74,9 +81,43 @@ let
       rev = "v${version}";
       hash = "sha256-LVLEn+e7c8013pwiLzJiiIObyrlbBHYaioO/SWbItPQ=";
     };
-    });
+  });
+
+  cutlass = fetchFromGitHub {
+    owner = "NVIDIA";
+    repo = "cutlass";
+    rev = "v3.0.0";
+    sha256 = "sha256-YPD5Sy6SvByjIcGtgeGH80TEKg2BtqJWSg46RvnJChY=";
+  };
+
+  cuda-common-redist = with cudaPackages; [
+    cuda_cccl # cub/cub.cuh
+    libcublas # cublas_v2.h
+    libcurand # curand.h
+    libcusparse # cusparse.h
+    libcufft # cufft.h
+    cudnn # cudnn.h
+  ];
+
+  cuda-native-redist = symlinkJoin {
+    name = "cuda-native-redist-${cudaVersion}";
+    paths = with cudaPackages; [
+      cuda_cudart # cuda_runtime.h cuda_runtime_api.h
+      cuda_nvcc
+    ] ++ cuda-common-redist;
+  };
```

Comment:
I don't remember if I asked before, but did you check if this works without `symlinkJoin`? The use of `symlinkJoin` for cuda stuff is an unfortunate artifact of certain packages' build scripts (e.g. tensorflow and bazel) which has many adverse side effects

Reply:
`buildInputs` and `nativeBuildInputs` doesn't need symlinkJoin, but I guess `-Donnxruntime_CUDA_HOME=${cuda.cuda-redist}` requires a symlinkJoin to pass a path to CMake.

Reply:
I don't find setting `-Donnxruntime_CUDA_HOME` necessary, CMake seems to pick it up just fine. I've removed both `symlinkJoin` uses.